app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(cp: str) -> None:
+    parts = cp.split(" ")
```

Comment:
You can simplify the unpacking in the way Valentyn wrote to you earlier.
command, source, destination = command.split()